filewatcher/filewatcher.go

```diff
@@ -132,9 +132,11 @@ var (
 )
 
 // Config defines the config to be used in New function.
+//
+// Can be deserialized from YAML.
 type Config struct {
 	// The path to the file to be watched, required.
-	Path string
+	Path string `yaml:"path"`
 
 	// The parser to parse the data load, required.
 	Parser Parser
```

Comment:
Since this field, which is required, cannot be deserialized from yaml, does it actually make sense to deserialize this config from yaml? how do we use this config in the yaml right now?

Reply:
I guess with two-step `baseplate.New` we no longer have this problem.